data/settings/icon-grid-th.json.in

```diff
@@ -25,6 +25,17 @@
     "net.sourceforge.Audacity.desktop",
     "org.gimp.Gimp.desktop"
   ],
+  "eos-folder-curiosity.directory" : [
```

Comment:
You need to call out the new folder in the "desktop" section.  Ditto for Vietnamese.

Reply:
Fixed and force pushed.